bluepyopt/ephys/evaluators.py

```diff
@@ -216,6 +216,17 @@ def evaluate_with_lists(self, param_list=None, target='scores'):
 
         return self.objective_list(obj_dict)
 
+    def set_neuron_variables_and_evaluate_with_lists(
+        self, param_list=None, target='scores'
+    ):
+        """Set neuron variables and run evaluation with lists.
+
+        Setting the neuron variables is necessary when using ipyparallel,
+        since the new subprocesses have pristine neuron.
```

Comment:
maybe NEURON to destinguish from the neuron model ?

Reply:
Done in latest commit